utils/jwt.ts

```diff
@@ -9,19 +9,48 @@ export function genJwt(id: string): string {
   return jwt.sign({id}, process.env.TOKEN_SECRET, { expiresIn: '7d' });
 }
 
-export function validateJwt(req: Request, res: Response, next: NextFunction) {
-  const authHeader = req.headers.authorization;
-  const token = authHeader && authHeader.split(' ')[1];
+interface validateJwtOptions {
+  req: Request;
+  res: Response;
+  next: NextFunction;
+  token?: string;
+  key: string;
+}
 
+function validateJwtToken({req, res, next, token, key}: validateJwtOptions) {
   if (token == null) return res.sendStatus(401).end();
 
   jwt.verify(token, process.env.TOKEN_SECRET as string, (err: VerifyErrors, decoded: any) => {
     if (err) {
       logger.error(err);
-      return res.sendStatus(403).end();
+      return res.status(403).json({key: true}).end();
```

Comment:
you need brackets around the variable `key` inorder for it to resolve in the object

```suggestion
      return res.status(403).json({[key]: true}).end();
```